src/index.html

```diff
@@ -6,9 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo-container">
+        <a class="logo_link" href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="list__items">
+          <li class="list__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="list__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
```

Comment:
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/7f00ff7e-b65f-4c23-962a-3b16cc15101b)